src/test/regress/expected/citus_non_blocking_split_shards.out

```diff
@@ -332,14 +341,24 @@ SELECT shard.shardid, logicalrelid, shardminvalue, shardmaxvalue, nodename, node
             ORDER BY 1, 2;
      relname     |         Constraint         |                                 Definition
 ---------------------------------------------------------------------
+ sensors_8981000 | fkey_table_to_dist_8981000 | FOREIGN KEY (measureid) REFERENCES colocated_dist_table_8981003(measureid)
```

Comment:
it would be good to force a cleanup after the shard splits to not change the intent of the tests

Reply:
Fixed.